### PR TITLE
add kubernetes server/ingress/logging config to repo #1226

### PR DESCRIPTION
**Pull request by tbeerman**
add kubernetes server/ingress/logging config to repo #1226

### CODE REVIEW

File: etc/docker/kubernetes/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+========================================
+Setting up Rucio in a Kubernetes cluster
+========================================
+
+
+Ingress configuration
+---------------------
+Depending on the platform hosting the Kubernetes cluster a variety of ingress controllers a available. The cluster currently used only supports Traefik so this readme only describes this but any other ingress controllers should be as well.
```
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: a available -> are available.

Comment (bari12) [reply etc/docker/kubernetes/README.rst]: should be as well -> should work as well??

File: etc/docker/kubernetes/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+First, to be able to serve https request a valid certificate/key is needed. Those can have to be added to the cluster as a secret::
```
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: can have -> have

File: etc/docker/kubernetes/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    $ kubectl create secret tls tls-secret --key tls.key --cert tls.crt --namespace=kube-system
+
+Next, the provided traefic.toml configuratio file has to be adeded to the cluster::
```
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: configuratio -> configuration
adeded -> added

File: etc/docker/kubernetes/README.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+    $ kubectl create configmap logging-configmap --from-file=ingress/traefik.toml --namespace=kube-system
+
+Then the daemonset configuration can be applied to cluster::
```
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: to -> to the

File: etc/docker/kubernetes/README.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    $ kubectl apply -f ingress/traefik-ds.yaml
+
+This will create the traefik daemonset and mount the certificate and configuration file. The daemonset works with a node selector, i.e., the ingress controller will only be run on nodes that the are labeled. You can label a node with::
```
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: that the -> that

**Review by bari12 — DISMISSED** [review]

File: etc/docker/kubernetes/README.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+After that the ingress controller should be up and running. You can check by trying to access the dashboard at: http://<node-name>:8080/dashboard/
+
+
+Starting a server instance
+--------------------------
+
+A server deployment is available under server/deployment.yaml. This will start up two pods running Rucio without TLS, expose port 80 as a service and add a new rule to the ingress with TLS termination.
+But before applying this deployment a valid rucio.cfg has to be added to the cluster::
+
+    $ kubectl create secret generic server-cfg --from-file=rucio.cfg
```
Comment (vingar) [inline etc/docker/kubernetes/README.rst:35]: rucio.cfg is missing in the repo. Would be good to add a reference one or the info about it

File: etc/docker/kubernetes/README.rst [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    $ kubectl create configmap logging-configmap --from-file=ingress/traefik.toml --namespace=kube-system
+
+Then the daemonset configuration can be applied to the cluster::
+
+    $ kubectl apply -f ingress/traefik-ds.yaml
+
+This will create the traefik daemonset and mount the certificate and configuration file. The daemonset works with a node selector, i.e., the ingress controller will only be run on nodes that are labeled. You can label a node with::
+
+    $ kubectl label node <node-name> role=ingress
+
```
Comment (vingar) [inline etc/docker/kubernetes/README.rst:25]: it's not clear for me. Would you have one example or latter in the documentation ?

File: etc/docker/kubernetes/README.rst (hunk 6 above)
Comment (tbeerman) [reply etc/docker/kubernetes/README.rst:35]: the reference one is rucio.cfg.template in etc/

File: etc/docker/kubernetes/README.rst (hunk 7 above)
Comment (tbeerman) [reply etc/docker/kubernetes/README.rst:25]: what's not clear about it? there is an example.

Comment (vingar) [reply etc/docker/kubernetes/README.rst:25]: sorry -- I'm learning kubernetes at the same time.   `kubectl get nodes` gives the list of nodes on the cluster.

**Comment by vingar** [comment]
@bari12 you're blocking this PR. Dismiss review ?

**Comment by bari12** [comment]
Oh I didn't realise. It's dismissed.

**Review by vingar — APPROVED** [review]

**Pull request merged**